app/javascript/components/EventForm/index.js

```diff
@@ -1,14 +1,12 @@
 import React from 'react'
 import { Field } from 'redux-form'
-import moment from 'moment-timezone'
 import R from 'ramda'
-import Geosuggest from 'react-geosuggest'
 import AutoComplete from 'material-ui/AutoComplete'
 import DatePicker from 'material-ui/DatePicker'
 import TimePicker from 'material-ui/TimePicker'
 
 import Callout from 'components/Callout'
-import Loading from 'components/LoadingIcon'
+import { LocationField } from 'lib/fields'
```

Comment:
no components in `lib`. Those should only be helper functions that are truly shared in many places.

Reply:
Good point. I'll extract `LocationField` out as a separate component.